azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/configuration/SpringConfiguration.java

```diff
@@ -17,7 +19,7 @@
     private Deployment deployment;
 
     public Boolean isPublic() {
-        return isPublic;
+        return BooleanUtils.isTrue(isPublic);
```

Comment:
Can we simple convert isPublic from Boolean to boolean

Reply:
Leave it Boolean for it may break validation in config mojo, if there is no break, will convert them to boolean in later pr.